apps/web/app/[boardUuid]/page.tsx

```
@@ -0,0 +1,45 @@
+'use client';
+
+import {useParams} from 'next/navigation';
+import {Map} from '@components/Map/Map';
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)